app/server.R

```diff
@@ -783,25 +784,41 @@ server <- function(input, output, session) {
         )
       )
     })
+  })
+
+  workflowCardsFiltered <- reactive({
+    dat <- workflowCards()
     # Filter by date
     dat <- Filter(\(w) {
       parse_date_tz(w$data$submission) >= parse_date_tz(paste(input$runs_date[1], "00:00:00")) &&
       parse_date_tz(w$data$submission) <= parse_date_tz(paste(input$runs_date[2], "23:59:00"))
     }, dat)
     # Filter by status
-    if (!is.null(input$workStatus)) {
+    if (!rlang::is_empty(input$workStatus)) {
       dat <- Filter(\(w) {
         w$data$status %in% input$workStatus
       }, dat)
     }
     # Filter by workflow name
     if (nzchar(input$workName)) {
       dat <- Filter(\(w) {
-        w$data$workflow_name == input$workName
+        w$data$workflow_name %in% input$workName
```

Comment:
Am I understanding right that this addresses #112?

Reply:
No. The `%in%` operator just changes behavior so that `input$workName` can be a vector instead of a string with `==`. It works for now, but it's not needed for the current `input$workName`, because the input is a single string. I think this is leftover from when I was testing the many inputs style box where the user could put in > 1 workflow name. I'll undo this change to make it more clear